style.css

```diff
@@ -527,6 +571,45 @@ article h1 {
   margin-top: 25px;
 }
 
+form {
+  margin-top: -30px;
+  display: flex;
+  margin-right: 9%;
+}
+
+.footer li input {
+  border: 1px solid #cfd8dc;
+  height: 48px;
+  width: 312px;
+  margin-top: 10px;
+  border-radius: 8px;
+  padding: 15px;
+}
+
+.footer li textarea {
+  border: 1px solid #cfd8dc;
+  height: 180px;
+  width: 312px;
+  margin-top: 10px;
+  border-radius: 8px;
+  padding: 15px;
+}
+
+.footer li button {
+  border: 1px solid #fff;
+  padding: 12px 16px;
+  border-radius: 8px;
+  font-family: "poppins", sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  color: #6070ff;
+  transition: all 0.2s ease;
+}
```

Comment:
- [ ] Please Implement the button interactions such as hover and pressed etc.. as it is required for us to match the figma design as close to as possible. here is the button interactions from the figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/212909944-4172560b-1ad7-43bf-94d6-8eda0e07ac02.png)